yarn.lock

```diff
@@ -1026,6 +1017,11 @@ caw@^2.0.1:
     tunnel-agent "^0.6.0"
     url-to-options "^1.0.1"
 
+cbor-web@^7.0.6:
```

Comment:
Where do we use this?

Reply:
Not sure, but it is probably some dependency of another dependency we are using.